spec/models/actor_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
+
+RSpec.describe Actor, type: :model do
+  describe "relationships" do
+    it { should have_many :movie_actors}
+    it { should have_many(:movies).through(:movie_actors) }
+  end
+end
```

Comment:
Your association tests look good, but we'd love to see model tests for each of your instance or class methods!